core/app/models/spree/store_credit_event.rb

```diff
@@ -33,16 +43,8 @@ def display_event_date
     end
 
     def display_action
-      case action
-      when Spree::StoreCredit::CAPTURE_ACTION
-        Spree.t('store_credit.captured')
-      when Spree::StoreCredit::AUTHORIZE_ACTION
-        Spree.t('store_credit.authorized')
-      when Spree::StoreCredit::ALLOCATION_ACTION
-        Spree.t('store_credit.allocated')
-      when Spree::StoreCredit::VOID_ACTION, Spree::StoreCredit::CREDIT_ACTION
-        Spree.t('store_credit.credit')
-      end
+      return "" if NON_EXPOSED_ACTIONS.include?(action)
```

Comment:
I thought an empty string would be a sensible default for a display method but nil is probably better here.